static/js/tippyjs.js

```diff
@@ -158,4 +161,33 @@ export function initialize() {
         hideOnClick: true,
         theme: "light-border",
     });
+
+    // compose box buttons popover shown on mobile widths.
+    delegate("body", {
+        target: ".compose_mobile_button",
+        placement: "top",
+        onShow(instance) {
+            instance.setContent(
+                render_mobile_message_buttons_popover_content({
+                    is_in_private_narrow: narrow_state.narrowed_to_pms(),
+                }),
+            );
+
+            $(instance.popper).on("click", instance.hide);
```

Comment:
Another issue not apparent in this is we should be using `one` instead of `on` here. Since `instance.popper` can be same element as I noticed.